src/containers/other/NFTView/elements/Stats/Stats.tsx

```diff
@@ -43,6 +49,7 @@ export const Stats: React.FC<StatsProps> = ({
 	bids,
 	isLoading,
 }) => {
+	const isMounted = useRef<boolean>();
```

Comment:
I believe this was the culprit for the react state update on unmounted component console warning